src/edu/duke/oit/vivo/webapp/search/documentBuilding/PersonCustomFields.java

```diff
@@ -76,11 +76,12 @@
           + " prefix obo: <http://purl.obolibrary.org/obo/> \n"
           + " PREFIX vcard: <http://www.w3.org/2006/vcard/ns#> \n"
           + " PREFIX duke: <http://vivo.duke.edu/vivo/ontology/duke-extension#> \n"
-          + " prefix vitro: <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#a> \n";
+          + " prefix vitro: <http://vitro.mannlib.cornell.edu/ns/vitro/0.7#a> \n"
+          + " PREFIX dukecv: <http://vivo.duke.edu/vivo/ontology/duke-cv-extension#> \n";
  
   private static final String query = 
       prefix 
-      + "SELECT ?alternateId ?title ?primaryEmail ?netid ?officehours ?profileURL \n"
+      + "SELECT ?alternateId ?title ?primaryEmail ?netid ?officehours ?profileURL ?interestsOverview ?teachingOverview ?administrativeOverview ?clinicalOverview ?positionsOverview \n"
```

Comment:
hi @jimwood - these look like the new fields that are part of the next phase of the project. I assume they won't cause any issues.